add_clip_ui.ui

```diff
@@ -43,11 +43,14 @@
    </property>
   </widget>
   <widget class="QRadioButton" name="newButton">
+   <property name="enabled">
+    <bool>true</bool>
```

Comment:
Le 15/06/2015 12:50, Julian Jarecki a écrit :
>
> In add_clip_ui.ui 
> <https://github.com/Vampouille/superboucle/pull/22#discussion_r32408695>:
>
> > @@ -43,11 +43,14 @@
> >     </property>
> >    </widget>
> >    <widget class="QRadioButton" name="newButton">
> > +   <property name="enabled">
> > +    <bool>true</bool>
>
> I'm actually not editing the .ui files manually, but with Qt Designer, 
> so it must have put the tag there.
>
You have a little button on QT designer to restore default value (and 
remove it from xml). Button is near values on the right.